doc/clang.txt

```diff
@@ -227,6 +227,16 @@ g:clang_use_path                        *g:clang_use_path*
     Default: >
         let g:clang_use_path = 1
 <
+g:clang_verbose_menu                        *g:clang_verbose_menu*
+        Whether to display detailed information in the menu (like
+        Youcompleteme).
+        For example, the output of '.push_back' after an 'std::vector<int>'
+        would be when set 1 >
+        push_back(const_reference __x) f [O] void
+<
+          Default: >
+          let g:clang_verbose_menu = 0
+
```

Comment:
```
g:clang_verbose_pmenu                        *g:clang_verbose_pmenu*
          Control the verbose level of popup menu, which is similar to Youcompleteme.
          For example, the popup menu would show 'push_back(const_reference __x) f [O] void'
          instead of 'push_back' if is set to 1.
     Default: >
          let g:clang_verbose_pmenu = 0
<
```